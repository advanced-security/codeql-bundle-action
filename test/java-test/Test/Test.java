class Test {
  Test() {}
  String source() { return "untrusted data"; }

  void sink(String arg) {}

  void test() {
    String data = source();
    sink(data);
  }
}