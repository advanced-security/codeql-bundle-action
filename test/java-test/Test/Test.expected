| Test.java:9:10:9:13 | data | Found flow from $@ | Test.java:8:19:8:26 | source(...) | source |